feature-configs/cn-ran-overlays/all-in-one/site.1.fqdn/networks/sriov-network-mh.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: mh-net-u-site-1-fqdn-worker1
+  namespace: openshift-sriov-network-operator
+spec:
+  ipam:  |
+    {
+    }
+  networkNamespace: mh-net-ns-site-1-fqdn-worker1
+  resourceName: mh_u_site_1_fqdn_worker1
```

Comment:
Can this have the same syntax of the other names and be mh-u-site-1... ?

Reply:
No, dashes in the resourceName are causing failures

Reply:
thanks, I was not soure about that.